clients/src/main/java/org/apache/kafka/clients/admin/UpdateFeaturesOptions.java

```diff
@@ -26,4 +26,14 @@
  */
 @InterfaceStability.Evolving
 public class UpdateFeaturesOptions extends AbstractOptions<UpdateFeaturesOptions> {
+    private boolean dryRun = false;
```

Comment:
Looking at the other admin client things, it seems like we should probably name this `validateOnly` to be consistent with everything else. AlterClientQuotas, AlterConfigs, CreatePartitions, CreateTopics, all use `validateOnly` and not `dryRun`.

(I guess this is a tiny KIP revision... but not too bad, right?)